examples/linux/crash_loop_detection/crash_loop_detection.cpp

```diff
@@ -96,9 +69,44 @@ void Crash()
     memset((void*)(intptr_t)4, 123, 1);
 }
 
-int main()
+int main(int argc, char** argv)
 {
-    std::cerr << startCrashHandler() << '\n';
+    /*
+     * YOU MUST CHANGE THIS VALUE.
+     *
+     * This should point to your server dump submission port
+     * (labeled as "http/writer" in the listener configuration
+     * pane. Preferably, the SSL enabled port should
+     * be used.
+     */
+    std::string url("https://submit.backtrace.io/yolo/c5a3a1675dbcfa1d12fd7efbc7bef9350874654e55ff86478115c8bc073f54dc/minidump");
```

Comment:
I think the URL should be different here - or at least with XXXX like in the previous example